.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will run tests using node and then publish a package to npm when a release is created
+# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: Publish to npm
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - run: npm ci
+      - run: npm test
```

Comment:
We don't have `ci` or `test` scripts defined in `package.json`, so these steps will probably fail.

Reply:
Good point! I’ve removed the testing step.